modules/insights-operator-configuring.adoc

```diff
@@ -3,26 +3,50 @@
 // * support/remote_health_monitoring/using-insights-operator.adoc
 
 
-:_mod-docs-content-type: PROCEDURE
+:_mod-docs-content-type: CONCEPT
 [id="insights-operator-configuring_{context}"]
 = Configuring Insights Operator
 
-You can configure Insights Operator to meet the needs of your organization. The Insights Operator is configured using a combination of the default configurations in the `pod.yaml` file in the Insights Operator `Config` directory and the configurations stored in the `support` secret in the `openshift-config` namespace. The `support` secret does not exist by default and must be created when adding custom configurations for the first time. Configurations in the `support` secret override the defaults set in the `pod.yaml` file.
+Insights Operator configuration is a combination of the default configuration and the configuration stored in either the support secret in the `openshift-config` namespace, or the *insights-config* ConfigMap in the `openshift-insights` namespace. The ConfigMap takes precedence over the support secret.
```

Comment:
I'd say whatever you decide to do with the `ConfigMap` object vs config map (general usage) try to be consistent across the PR.